mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+follow_imports = skip
+
+[mypy-ansible.module_utils.six.*]
+#follow_imports = skip
+ignore_missing_imports = True
+
+[mypy-resolvelib.*]
+#follow_imports = skip
+ignore_missing_imports = True
```

Comment:
Move this to `test/sanity/code-smell/mypy.ini` to keep it with the other mypy related files, and out of the root.